documentation/src/main/asciidoc/topics/con_session_externalization.adoc

```diff
@@ -0,0 +1,21 @@
+[id='session-externalization_{context}']
+= Session externalization
+
+{brandname} can provide external caches for applications built on Red Hat JBoss Enterprise Application Platform (EAP), Red Hat Build of Quarkus, Red Hat JBoss Web Server (JWS), and Spring.
```

Comment:
Since you name here the RH products I wonder if we should say Spring Boot instead or Spring Framework

Reply:
Thanks for catching this. I've added variables for the community names. I think Spring is OK here. If we wanted to be sticklers I think we could maybe say Spring Framework but I'm not sure that's needed.